dev_guide/new_app.adoc

```diff
@@ -483,6 +483,37 @@ from the global library:
 ====
 
 image::console_select_image.png["Select Builder Image"]
+
+[NOTE]
+Only ImageStreamTags which have the 'builder' tag listed in their annotations will appear in this list, as demonstrated here:
+
+----
+    {
+      "kind": "ImageStream",
+      "apiVersion": "v1beta3",
```

Comment:
@bparees I know this is only an example, but should this be "v1" instead of "v1beta3"?